rep-0147.rst

```diff
@@ -341,6 +343,33 @@ To send a series of goals a list of GlobalPositions can be used.
   nav_msgs/GlobalPosition[] waypoints
 
 
+Pose Reporting
+------------------
+
+An aerial vehicle can share its geographic pose via ``geographic_msgs/GeoPoseStamped`` [#geoposestamped]_.
+The position coordinates should follow WGS-84 [#wgs84]_.
+
+EGM2008 [#egm]_ can be assumed as the altitude datum unless otherwise stated.
+If and when EGM2020 is released, this REP can be updated.
+Take caution integrating GPS data and terrain data as they frequently use different datums.
+
+The orientation field follows REP 103's [#REP103]_ recommendation. The orientation is the transform from the 
+local tangent plane [#ltpc]_ to the body frame.
+
+A vehicle in level flight facing east will have a unit identity quaternion for its orientation.
+
+x,y,z=0, w=1
+
+.. image:: rep-0147/geopose-default-orientation.png
+
+Remember that per REP 103 [#REP103]_, when the orientation is decomposed to its yaw component,
+this yaw value is NOT equal to the traditional compass bearing.
+
+In the situation that the position is known but orientation it not known, or vise versa, 
+the unknown fields should be populated with ``NaN`` to avoid Null Island [#null_island]_.
```

Comment:
```suggestion
the unknown fields should be populated with ``NaN`` to avoid Null Island [#null_island]_.
It is up to the platform to decide at what accuracy threshold the data will stop being published
or switch to ``NaN`.
```

Thoughts on this for addressing accuracy?